templates/cases/case_read.haml

```diff
@@ -151,9 +151,31 @@
                   [[ key ]]
                 : [[ val ]]
 
+        - for pod in pods
+            %div{
+              ng-controller:"={ pod.pod_type.controller }",
+              ng-init:'init && init(={ pod.config.index }, ={ case_id }, ={ pod.config_json })',
+              "={ pod.pod_type.directive }":"true"
+            }
```

Comment:
So after trying things out, I'm not sure how easily we'd be able to use directives as elements. The directive names depend on the context given to the template, and I can't seem to find a way of providing tags with names that depend on the context with hamlpy. Maybe it would be easier to stick with attributes?